en/docs/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup.md

```diff
@@ -87,6 +87,17 @@ This section involves setting up the Key Manager node and enabling it to work wi
      [[apim.throttling.url_group]]
      traffic_manager_urls = ["tcp://Traffic-Manager-2-host:9611"]
      traffic_manager_auth_urls = ["ssl://Traffic-Manager-2-host:9711"]
+
+     [[event_listener]]
```

Comment:
hopefully these are included in the config catalog

Reply:
this was came from Identity server configs since they didn't have a config catalog we might need to include into our catalog
@ruthryi